release/llm_tests/serve/probes/test_basic.py

```diff
@@ -304,19 +304,15 @@ async def test_logprobs(
     )
 
     response = await deterministic_query.query(model, stream, **params)
+
     response = response.full_dict()
     for resp in response:
-        running_str = ""
         for logprob in resp["logprobs"]["content"]:
             assert len(logprob["top_logprobs"]) == num_logprobs
             assert list(logprob["token"].encode()) == logprob["bytes"]
-            # Special tokens that will not be a part of the response content
```

Comment:
so vllm V1 on some models (depending on the tokenizer) don't have this property to not return the decoded unicode characters. In V0 this was the case. If this is an issue from user's perspective it needs to be addressed from the engine side.